### DISCUSSION TITLE
In-Network File - Optional Fields for Negotiated Rate Based on Percentage

### DISCUSSION THREAD

**Discussion by ghost** [CMSgov/price-transparency-guide#87] [Q&A] [answered]
Per the earlier discussion, If negotiated rate is based on percentage of charge--then how would this be reported in the in-network file? #23 reply by @shaselton-usds 

To provide context, we initially thought through possible implementation scenarios like this:

{  
  "reporting_entity_name": "medicare",  
  "reporting_entity_type": "medicare",  
  ...
  "in_network": [{     
    "negotiation_arrangement": "reference based",    
    "name": "Knee Replacement",    
    ...
    "negotiated_rates": [{      
       "reference_based_percentage": "103",
       "reference_based_standard": "medicare",
       "weighted_average_price": 1000.00, // drg base rate = (location, wage index, hospital type, etc). Average location variable number, Average wage index number, Average hospital type number
     ....
}
It added more optional attributes which drove up complexity.

It looks like we might need to technically add "**referenced based**" to one of the possible values for negotiation_arrangement to signify the arrangement, yes? If so, the documentation will be added to include referenced based (it might have been an oversight with the initial publication).

**Comment by ghost** [comment]
The addition of a optional field for "**weighted_average_price**" would be useful in the case where a percentage based rated is used to allow the understanding of what the rate was. I do not think we would need to list the "**reference_based_percentage**" as required, but could be optional since the value could be found by reference_based_percentage = rate / weighted_average_price.

**Comment by tdfow** [comment]
If the basis is percentage of charge, then would the 'referenced_based_standard' be "charge"?    Are you suggesting another object for this scenario instead of the negotiated_price or negotiated_rates?  I am not following the example provided to address the percentage of charge scenario.   I assume there still needs to be the negotiated_rate published--but it isn't noted in the scenario.   It would seem that 'referenced_based_charge_basis' type field would allow the amount used in the calculation of the negotiated_rate; but not following how the above addresses the percentage of charge scenario, as how is it expected to obtain a weighted average for every item/service?

**Comment by shaselton-usds** [comment, marked as answer]
Wrapping up old threads -- hopefully the new additions to support [percentage of bill charges](https://github.com/CMSgov/price-transparency-guide/pull/393) answer this question. Otherwise, if the negotiation is a referenced based negotiation like a percentage of Medicare, then use the base-rate and multiple the percentage to get to a dollar amount. Let me know otherwise.
